CUEstion.BLL/QuestionManagerService.cs

```diff
@@ -140,9 +147,9 @@ public void UpdateQuestion(QuestionDTO questionDto)
 			foreach (var tag in questionDto.Tags)
 			{
 				//rewrite case-insensitive
-				if (context.Tags.Select(t => t.Name).Any(t => EF.Functions.Like(t, tag)))
+				if (context.Tags.Any(t => t.Name.ToLower() == tag.ToLower()))
```

Comment:
You can find the first(and only) tag before the if statement, using something like var tagItem = context.Tags.First(t => t.Name.ToLower() == tag.ToLower()). And then check if tagItem == null. If not - question.Tags.Add(tagItem);